system/Database/BaseConnection.php

```diff
@@ -854,6 +854,14 @@ public function table($tableName)
         return new $className($tableName, $this);
     }
 
+    /**
+     * Returns a new instance of the BaseBuilder class with a cleared FROM clause.
+     */
+    public function newQuery(): BaseBuilder
+    {
+        return $this->table(uniqid())->from([], true);
```

Comment:
I think the cost of calling `uniqid()` is wasteful.
The table name will be removed by the next `from()` method.
It seems there is no need for uniqueness.

Reply:
Yes and no.
The table name specified in the constructor of the BaseBuilder class is stored in the ``$tableName`` property, which is then used in the Model. Yes, the dependence is weak, but who knows what else will come up with.

Suggest a temporary table name right here.
I will accept it immediately.

Reply:
This is a bit tricky, `','`. It can set empty string $tableName property.

Reply:
I like it